includes/Block.php

```diff
@@ -19,7 +19,9 @@
  *
  * @file
  */
+use \Wikia\Service\User\Permissions\PermissionsServiceAccessor;
 class Block {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "Block" has 56 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1448)